modules/nw-sriov-dpdk-example-intel.adoc

```diff
@@ -66,17 +66,20 @@ metadata:
   namespace: openshift-sriov-network-operator
 spec:
   networkNamespace: <target_namespace>
-  ipam: "{}" <1>
+  ipam: |-
+# ... <1>
   vlan: <vlan>
   resourceName: intelnics
 ----
-<1> Specify an empty object `"{}"` for the ipam CNI plug-in. DPDK works in userspace mode and does not require an IP address.
+<1> Specify a configuration object for the ipam CNI plug-in as a YAML block scalar. The plug-in manages IP address assignment for the attachment definition.
 +
 [NOTE]
 =====
 See the `Configuring SR-IOV additional network` section for a detailed explanation on each option in `SriovNetwork`.
 =====
 +
+An optional library, app-netutil, provides several API methods for gathering network information about a pod from within a container running within that pod.
```

Comment:
s/running within/running on (or similar) to avoid the repeated _within_??

Or, remove the first _within_?
_gathering network information about a pod from a container_